templates/faq/embedded-faq.html

```diff
@@ -0,0 +1,252 @@
+{# List of questions #}
+{% set faq_questions_en = [
+    "editing/map-errors",
+    "map/can-not-find-position",
+    "map/cant-download-maps",
+    "map/search-cannot-find-a-place",
+    "app/crash",
+    "bookmarks/how-to-export",
+    "bookmarks/how-to-import"
+] %}
+
+{% set faq_languages = ["ru", "de", "es", "fr", "pl", "pt", "pt-BR", "tr", "uk", "zh-Hans"] %} {# Predefined list of translation languages #}
+
+<div class="faq">
+    <h2 class="title" lang="en">FAQ</h2>
+    <h2 lang="ru">Часто задаваемые вопросы (FAQ)</h2>
+    <h2 lang="de">FAQ</h2>
+    <h2 lang="es">FAQ</h2>
+    <h2 lang="fr">FAQ</h2>
+    <h2 lang="pt">Perguntas frequentes</h2>
+    <h2 lang="pt-BR">Perguntas frequentes</h2>
+    <h2 lang="pl">FAQ</h2>
+    <h2 lang="tr">SSS</h2>
+    <h2 lang="uk">Часті питання</h2>
+    <h2 lang="zh">常见问题</h2>
+
+    <ul>
+    {% for faq_slug in faq_questions_en %}
+        {% set faq_page_en = get_page(path = "faq/" ~ faq_slug ~ "/index.md") %}
+        <li class="link">
+            <a lang="en" href="#{{ faq_slug | replace(from="/", to="-") }}">{{ faq_page_en.title }}</a>
+            {# Add translated pages #}
+            {% for lang in faq_languages %}
+                {% set translated_faq_page = faq_page_en.translations | filter(attribute="lang", value=lang) | first %}
+                {% if not translated_faq_page %}
+                    {{ throw(message="Page 'faq/" ~ faq_slug ~ "/index.md' is not translated into '" ~ lang ~ "'") }}
+                {% endif %}
+                <a lang="{{ lang | replace(from="-Hans", to="") }}" href="#{{ faq_slug | replace(from="/", to="-") }}">{{ translated_faq_page.title }}</a>
+            {% endfor %}
+        </li>
+    {% endfor %}
+    </ul>
+
+    <h4 lang="en">
+        For more information please visit our web-site
+        <a href="https://organicmaps.app/">organicmaps.app</a>
+    </h4>
+    <h4 lang="ru">
+        Больше информации на нашем сайте
+        <a href="https://organicmaps.app/ru/">organicmaps.app</a>
+    </h4>
+    <h4 lang="de">
+        Für weitere Informationen besuchen Sie bitte unsere Website
+        <a href="https://organicmaps.app/">organicmaps.app</a>
+    </h4>
+    <h4 lang="es">
+        Para más información, visite nuestro sitio web
+        <a href="https://organicmaps.app/">organicmaps.app</a>
+    </h4>
+    <h4 lang="fr">
+        Pour plus d'informations, veuillez visiter notre site web
+        <a href="https://organicmaps.app/">organicmaps.app</a>
+    </h4>
+    <h4 lang="pt">
+        Para mais informações, visite o nosso sítio na web
+        <a href="https://organicmaps.app/">organicmaps.app</a>
+    </h4>
+    <h4 lang="pt-BR">
+        Para mais informações, visite o nosso sítio web
+        <a href="https://organicmaps.app/">organicmaps.app</a>
+    </h4>
+    <h4 lang="pl">
+        Aby uzyskać więcej informacji, odwiedź naszą stronę internetową
+        <a href="https://organicmaps.app/">organicmaps.app</a>.
```

Comment:
```suggestion
        <a href="https://organicmaps.app/pl/">organicmaps.app/pl/</a>.
```

Reply:
I used for-loop to generate links in format `organicmaps.app/{{ lang }}/`